packages/cozy-realtime/README.md

```diff
@@ -40,68 +40,60 @@ or
 
 ## API
 
-#### `subscribeAll(config, doctype, parse)`
+#### `subscribe(config, doctype, options)`
 
-This method allow you to subscribe to realtime for all documents of a provided doctype. Here are the parameters:
+This method is exactly the same working as the previous `subscribeAll()` function but to listen only one document. Here are the parameters:
```

Comment:
I would not compare this method with the former method since it is unlikely that the person reading the doc is familiar with the old one.

Reply:
Oops I forgot to remove that, thanks 👍